certz/certz.proto

```diff
@@ -480,6 +516,7 @@ message Entity {
     CertificateChain certificate_chain = 3;
     CertificateChain trust_bundle = 4;
     CertificateRevocationListBundle certificate_revocation_list_bundle = 5;
+    AuthenticationPolicy authentication_policy = 6;
```

Comment:
message UploadRequest contains "repeated Entity". In each Entity, we can have oneof the above 4 items. Is it expected that all the items in the 'entities' list are of different entity types? In other words, can the UploadRequest contain at max 4 'message Entity'?
Though it is possible to create UploadRequest like [certificate_chain, certificate_chain, authentication_policy, authentication_policy] it is wrong. Maybe it can be cleared in the spec that each message in the list must be of a different type. If not, return an appropriate error.

Reply:
The scenario you have described is possible but while it is strange and seems invalid it there is no problem with it as the last instance of particular type of entity will be made 'active'. 

It is similar to the following piece of C++ code:

```
int a = 0;
a = 1;
a = 2;
```

What counts is that eventually `a == 2` and depending on the implementation the intermediate steps might be ignored and not performed.

Similar effect can be achieved by sending multiple upload request messages over one `Rotate()` stream - what counts is the last one as it will be made 'active' once the `Finalize` message is sent.

Reply:
Thank you. This is clear